### ISSUE TITLE
Integrate builds

### ISSUE THREAD

**Issue by am7c7** [KonsecGmbH/intellij-onedev-plugin#11]
It would be nice to see build status of the current project. Show status of running build, finished builds etc
Logs and direct watching of running build logs would be very cool.

**Comment by alex-sc** [comment]
@am7c7 Did you check https://github.com/KonsecGmbH/intellij-onedev-plugin/actions

**Comment by am7c7** [comment]
No, I mean the onedev builds. Onedev can run CI jobs. But this issue is just a reminder for future improvements.

**Comment by alex-sc** [comment]
@am7c7 Got you. This plugin currently uses solely the Task API (some docs here https://intellij-support.jetbrains.com/hc/en-us/community/posts/207098975-Documenation-about-Task-Api)

Of course it's possible to add more features out of the Task API scope. It's better to find an existing plugin which does something similar for reference

**Comment by am7c7** [comment]
https://plugins.jetbrains.com/plugin/6110-jenkins-control
Source code: https://github.com/mcmics/jenkins-control-plugin
